src/app/api/crud.py

```diff
@@ -14,18 +14,18 @@ async def get(entry_id: int, table: Table):
     return await database.fetch_one(query=query)
 
 
-async def fetch_all(table: Table, query_filters: Optional[List[Tuple[str, Any]]] = None):
+async def fetch_all(table: Table, query_filters: Optional[Dict[str, Any]] = None):
     query = table.select()
-    if isinstance(query_filters, list):
-        for query_filter in query_filters:
-            query = query.where(getattr(table.c, query_filter[0]) == query_filter[1])
+    if isinstance(query_filters, dict):
+        for queryFilterKey, queryFilterValue in query_filters:
```

Comment:
Nitpicking but could you put your variables in snake_case?

Reply:
Done